ocf/ocf.go

```diff
@@ -55,7 +55,12 @@ type Decoder struct {
 
 // NewDecoder returns a new decoder that reads from reader r.
 func NewDecoder(r io.Reader) (*Decoder, error) {
-	reader := avro.NewReader(r, 1024)
+	return NewDecoderWithConfig(r, avro.DefaultConfig)
+}
+
+// NewDecoderWithConfig returns a new decoder that uses the provided config.
+func NewDecoderWithConfig(r io.Reader, conf avro.API) (*Decoder, error) {
```

Comment:
I would prefer you follow the options pattern as is done with the OCF encoder.

Reply:
Addressed. Tried to mirror the `Encoder`. Let me know if there's another option you'd prefer!